data/beta/field/copyright.yaml

```diff
@@ -3,4 +3,4 @@ datatype: text
 field: copyright
 phase: beta
 register: ''
-text: Copyright for the data in the register.
+text: The owner or copyright that protects the data in a register.
```

Comment:
We used it for the original address and postcode registers, and it covers licensing terms as well as copyright (which are different). I'll try and change the wording.